addons/website_sale/models/product_template.py

```diff
@@ -198,13 +198,16 @@ def _get_combination_info(self, combination=False, product_id=False, add_qty=1,
                 list_price = taxes.compute_all(combination_info['list_price'], pricelist.currency_id, quantity_1, product, partner)[tax_display]
             else:
                 list_price = price
+            combination_info['price_extra'] = self.env['account.tax']._fix_tax_included_price_company(combination_info['price_extra'], product.sudo().taxes_id, taxes, company_id)
```

Comment:
```suggestion
            combination_info['price_extra'] = self.env['account.tax']._fix_tax_included_price_company(combination_info['price_extra'], product_taxes, taxes, company_id)
```

Little simplification in multi-comp environment, use already filtered taxes for `_fix_tax_included_price_company` call (like it's done in the previous call earlier in the code)

Reply:
Thanks :+1: